tools/makefile-support/helpsystem-targets

```diff
@@ -0,0 +1,20 @@
+# Default prefix for commands in the main help command, e.g. "make build".
+# The FDM utility may change this to "fdm".
+HELP_CMD_PREFIX ?= make
+CMD = $(ARG1)
+
+# Run the main help command.
+help:
+	@CMD=$(CMD) REFORMAT_OPTIONS=$(REFORMAT_OPTIONS) HELP_CMD_PREFIX=$(HELP_CMD_PREFIX) ./tools/makefile-support/makehelp.sh
+
+# Print separators between help sections.
+.help-sep-1:
+	@printf "\036"
+.help-sep-2:
+	@printf "\036"
+.help-sep-3:
+	@printf "\036"
+.help-sep-4:
+	@printf "\036"
+.help-sep-5:
+	@printf "\036"
```

Comment:
I couldn't get `make` to repeat the same target no matter what I tried, so I just added multiple targets that do the same thing.  See `makehelp.sh` for where they're used.